### ISSUE TITLE
Explore options for image alt text

### ISSUE THREAD

**Issue by bdezray** [psu-libraries/scholarsphere#1622]
The best approach would be to use Uppy to display a method for applying alt text when an image file is uploaded. So, first step would be to determine if this is something that Uppy can do.

**Comment by ajkiessl** [comment]
Re: Where do we present the alt text?

Looks like the general rule is that if an image is an essential part of a larger experience, then there needs to be alt text describing it somewhere.  If the image is not displayed (only a download link) then the alt text should be associated with the image download link in some way.  An aria-label is sufficient.
